kali_sg.tf

```
@@ -65,6 +51,30 @@ resource "aws_security_group_rule" "ingress_from_anywhere_to_kali_via_allowed_po
   to_port           = each.value["to_port"]
 }
 
+# Allow unfettered access between Kali and Teamserver instances
+resource "aws_security_group_rule" "kali_egress_to_teamserver_instances" {
+  provider = aws.provisionassessment
+  for_each = toset(["tcp", "udp"])
+
+  security_group_id        = aws_security_group.kali.id
+  type                     = "egress"
+  protocol                 = each.key
+  source_security_group_id = aws_security_group.teamserver.id
+  from_port                = 5000
+  to_port                  = 5999
+}
+resource "aws_security_group_rule" "kali_ingress_from_teamserver_instances" {
```

Same issue as above.
```suggestion
resource "aws_security_group_rule" "kali_ingress_from_teamserver_instances_via_5000s" {
```

---

Please see commit 0a70a6e.